### DISCUSSION TITLE
[Icon Request] Social Media and Educational icon

### DISCUSSION THREAD

**Discussion by prame-dot-exe** [tailwindlabs/heroicons#1205] [Ideas]
Can you make social media icons. Example of Facebook, Instagram, Line etc. And if possible can you make the education icon like atom, or math (+, - , * , /), or physic. That's will make you not the same as another cuz no one make the educational icon but Social or the big company such as google or Microsoft will be necessary for many project. Thanks

**Comment by ionutcirja** [comment]
Pretty please.

**Comment by zefir-git** [comment]
Why burden Heroicons with brands/logos when https://simpleicons.org/ is specifically for this purpose
